fdbserver/Resolver.actor.cpp

```diff
@@ -446,20 +451,21 @@ ACTOR Future<Void> resolveBatch(Reference<Resolver> self,
 
 		// If shardChanged at or before this commit version, the proxy may have computed
 		// the wrong set of groups. Then we need to broadcast to all groups below.
-		stateTransactionsPair.first = toCommit && toCommit->haveLogsChanged();
+		bool logsChanged = applyResolverPrivateMutations && toCommit->haveLogsChanged();
```

Comment:
Should we be using "useResolverPrivateMutations" in this check (instead of "applyResolverPrivateMutations")? Or is there any guarantee that "req.txnStateTransactions" is not going to be empty if "toCommit->haveLogsChanged()" is true here?